lib/iris/tests/unit/analysis/test_Nearest.py

```diff
@@ -103,5 +103,39 @@ def test_nanmask(self):
         self.check_mode('nanmask')
 
 
+class Test_regridder(tests.IrisTest):
+    def check_mode(self, mode=None):
+        scheme = create_scheme(mode)
+
+        with mock.patch('iris.analysis.RectilinearRegridder',
```

Comment:
I'm not sure it's worth a cool million pedant points (more like half a dozen), but it'd be nice to have a comment here to explain what's going on - i.e. a tweaked version of the [comment for the interpolator](https://github.com/bjlittle/iris/blob/a9631b8c5746777f91a1825dff7eb158c083046d/lib/iris/tests/unit/analysis/test_Nearest.py#L72).